### PR TITLE
docs: fixed zh/response-rewrite.md format

### PR DESCRIPTION
**Pull request by liangliang4ward**
### What this PR does / why we need it:

fixed response-rewrite-docs format

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/plugins/response-rewrite.md [hunk 1]
```diff
@@ -23,19 +23,21 @@ title: response-rewrite
 
 ## 目录
 
-- [**response-rewrite**](#response-rewrite)
+- [**简介**](#简介)
```
Comment (spacewander) [inline docs/zh/latest/plugins/response-rewrite.md:26]: Maybe we can remove them as we already generate the table of content on the doc site?
https://apisix.apache.org/docs/apisix/plugins/redirect

@juzhiyuan 
What about your opinion?

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/plugins/response-rewrite.md (hunk 1 above)
Comment (juzhiyuan) [reply docs/zh/latest/plugins/response-rewrite.md:26]: Agree! We have more people using website 😉

Comment (liangliang4ward) [reply docs/zh/latest/plugins/response-rewrite.md:26]: > Agree! We have more people using website 😉

so? remove them?and also other docs?

Comment (juzhiyuan) [reply docs/zh/latest/plugins/response-rewrite.md:26]: Hi @liangliang4ward, I would prefer to keep this PR as current status and submit a new PR to remove all Outlines from markdown files, @liangliang4ward could you help to do that?

Comment (liangliang4ward) [reply docs/zh/latest/plugins/response-rewrite.md:26]: i'd like to

**Pull request merged**